src/twisted/internet/test/test_inlinecb.py

```diff
@@ -145,3 +147,181 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         self.assertIn("in erroring", f.getTraceback())
+
+
+
+class TranslateMe(Exception):
+    """
+    Sample exception type.
+    """
+
+
+
+class TranslateResult(Exception):
+    """
+    Sample exception type.
+    """
+
+
+
+class DontFail(Exception):
+    """
+    Sample exception type.
+    """
+
+    def __init__(self, actual):
+        Exception.__init__(self)
+        self.actualValue = actual
+
+
+
+class CancellationTests(TestCase):
+    """
+    Tests for cancellation of L{Deferred}s returned by L{inlineCallbacks}.
+    For each of these tests, let:
+        - G be a generator decorated with C{inlineCallbacks}
+        - D be a L{Deferred} returned by G
+        - C be a L{Deferred} awaited by G with C{yield}
+    """
+
+    def setUp(self):
+        """
+        Set up the list of outstanding L{Deferred}s.
+        """
+        self.thingsOutstanding = []
```

Comment:
This needs a more descriptive name than `thing`.  Why not `deferredsOutstanding`?